python/nav/web/machinetracker/views.py

```diff
@@ -516,3 +517,18 @@ def netbios_do_search(request):
 
     info_dict.update(NBT_DEFAULTS)
     return render(request, 'machinetracker/netbios_search.html', info_dict)
+
+
+VALID_HELP_TAB_NAMES = ['ip', 'mac', 'switch', 'netbios']
```

Comment:
Please move this constant to the top of the file :)

Reply:
Done :)